Zend/tests/match/037.phpt

```diff
@@ -53,13 +53,13 @@ var_dump(match(3) {
 
 ?>
 --EXPECTF--
-string(%d) "UnhandledMatchError: Unhandled match value of type bool in %s037.php:4
+string(%d) "UnhandledMatchError: Unhandled match bool(true) in %s:%d
```

Comment:
Not a big fan of this var_dump-like formatting for error messages. I'd expect this to just say `true`, `1`, `1.0`, `"1"` to distinguish types.

More generally, we have a bunch of code that prints values in a "nice" way, e.g. https://github.com/php/php-src/blob/ae8647d9d3fa420e1116af02ac431772f81c9968/Zend/zend_exceptions.c#L494-L538 and https://github.com/php/php-src/blob/ae8647d9d3fa420e1116af02ac431772f81c9968/ext/reflection/php_reflection.c#L612-L643. I think it's time to consolidate something.

Reply:
Done